src/Microsoft.Health.Dicom.Core/Features/Store/Entries/DicomInstanceEntryReaderForSinglePartRequest.cs

```diff
@@ -5,14 +5,20 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
 using System.IO;
 using System.Net.Http.Headers;
+using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
 using EnsureThat;
+using Microsoft.Extensions.Options;
 using Microsoft.Health.Abstractions.Exceptions;
+using Microsoft.Health.Dicom.Core.Configs;
+using Microsoft.Health.Dicom.Core.Exceptions;
 using Microsoft.Health.Dicom.Core.Web;
+using NotSupportedException = System.NotSupportedException;
```

Comment:
Is there a name conflict?

Reply:
Yes, there is one that we created for some dicom things that are note supported.